celesta-sql/src/main/javacc/CelestaParser.jj

```diff
@@ -24,18 +24,18 @@ import ru.curs.celesta.score.validator.IdentifierParser;
 //CHECKSTYLE:OFF
 public class CelestaParser{
     public static Expr parseComplexFilter(String filter, IdentifierParser identifierParser) {
-         StringReader sr = new StringReader(filter);
-         try {
-                 CelestaParser parser = new CelestaParser(sr);
-                 try {
-                      return parser.sqlExpression(identifierParser);
-                 } catch (ParseException e) {
-                    throw new CelestaException("Invalid complex filter '%s': %s", filter, e.getMessage());
-                 }
-     } finally {
-         sr.close();
-     }
-   }
+        StringReader sr = new StringReader(filter);
```

Comment:
I see. We will definitely fix this try-with-resources issue in the next version of JavaCC!!!